tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1046,8 +1054,11 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
 
   Status FuseMatrixBiasAdd(HloInstruction *instr, HloInstruction *bias,
```

Comment:
Please comment the structure of instructions this function expects. E.g., the comment can be

```
Fuses a matrix bias into a cuBLAS call. 'instr' should be an Add instruction in the following form:
    Add(OptionalBitcast(OptionalSlice(gemm)), bias)
'gemm' is expected to be a cuBLAS custom_call.
```

Reply:
Resolved.